bitnami/elasticsearch/templates/_helpers.tpl

```diff
@@ -126,9 +126,18 @@ Create a default fully qualified coordinating name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "elasticsearch.coordinating.fullname" -}}
+{{/* If Kibana is enabled. Evaluated in the Kibana subchart */}}
+{{- if .Values.enabled -}}
```

Comment:
`.Values.enabled`?

Reply:
The same I replied in Juan's comment, it is because this function is evaluated in two charts, while in ES is `.Values.kibana.enabled` in the Kibana chart is `.Values.enabled`